frontend/src/app/components/tasque-card/tasque-card.component.html

```diff
@@ -36,8 +36,8 @@
     <div class="task-info">
       <span>{{ taskInfo.key }}</span>
       <tasque-avatar
-        *ngIf="taskInfo.author"
-        [user]="taskInfo.author"
+        *ngIf="taskInfo.assignees && taskInfo.assignees.length > 0"
```

Comment:
use `taskInfo.assignees?.length > 0`

Reply:
`*ngIf="taskInfo.assignees` is needed because we cannot write `taskInfo.assignees[0]` without this check.